autoload/ghpr_blame/app.vim

```diff
@@ -46,6 +46,9 @@ function! s:_extract_slug() dict abort
     if empty(m)
         let m = matchlist(out, printf('^https://\%%([^@/]\+@\)\?%s/\([^/]\+/[^/]\+\)\.git\n$', host))
     endif
+    if empty(m)
+        let m = matchlist(out, printf('^ssh://\%%([^@/]\+@\)\?%s/\([^/]\+/[^/]\+\)\n$', host))
+    endif
```

Comment:
Rather than adding new `matchlist()` call, could you modify the regex at L45 because `git@...` is just an abbreviation of `ssh://git@...`? I guess following should work.

```vim
let m = matchlist(out, printf('^\%(ssh://\)\=git@%s:\([^/]\+/[^/]\+\)\.git\n$', host))
```

Reply:
hmm...

`ssh://git@github.com/:owner/:repo` pattern doesn't contain trailing `.git`. It is too hard for me to build a regexp that matches as strict so I added new `matchlist()`.

Reply:
um, ok. I got the point. Then I think your suggestion would be acceptable rather than making the regex L45 more complex. Thank you.